File: solutions/problem_0019/C/Main.c
#include <stdio.h>

int main(){
    
    int valor_lido;
    int valor, qtdade_notas, valor_nota;

    printf("Digite o valor: \n");
    scanf("%d", &valor_lido);

    valor = valor_lido;
    
    valor_nota = 100;
    qtdade_notas = valor / valor_nota;
    valor = valor % valor_nota;
    printf("Notas de R$ %4d.00: %3d notas\n", valor_nota, qtdade_notas);

    valor_nota = 50;
    qtdade_notas = valor / valor_nota;
    valor = valor % valor_nota;
    printf("Notas de R$ %4d.00: %3d notas\n", valor_nota, qtdade_notas);

    valor_nota = 10;
    qtdade_notas = valor / valor_nota;
    valor = valor % valor_nota;
    printf("Notas de R$ %4d.00: %3d notas\n", valor_nota, qtdade_notas);

    valor_nota = 5;
    qtdade_notas = valor / valor_nota;
    valor = valor % valor_nota;
    printf("Notas de R$ %4d.00: %3d notas\n", valor_nota, qtdade_notas);

    valor_nota = 1;
    qtdade_notas = valor / valor_nota;
    valor = valor % valor_nota;
    printf("Notas de R$ %4d.00: %3d notas\n", valor_nota, qtdade_notas);

    return 0;
}

File: solutions/problem_0019/Python/main.py
#--------------------------------------#
#--- Elaborado por: Janaina Abib    ---#
#--- Data: 24/03/2024               ---#
#--------------------------------------#

def principal( ):
    
    valor_lido = float(input("Digite o valor: "))
    
    valor = valor_lido
    
    valor_nota = 100
    qtdade_notas = valor // valor_nota
    valor = valor % valor_nota
    print("Notas de R${:.2f}: {} nota(s)".format(valor_nota, qtdade_notas))

    valor_nota = 50
    qtdade_notas = valor // valor_nota
    valor = valor % valor_nota
    print("Notas de R${:.2f}: {} nota(s)".format(valor_nota, qtdade_notas))

    valor_nota = 10
    qtdade_notas = valor // valor_nota
    valor = valor % valor_nota
    print("Notas de R${:.2f}: {} nota(s)".format(valor_nota, qtdade_notas))

    valor_nota = 5
    qtdade_notas = valor // valor_nota
    valor = valor % valor_nota
    print("Notas de R${:.2f}: {} nota(s)".format(valor_nota, qtdade_notas))

    valor_nota = 1
    qtdade_notas = valor // valor_nota
    valor = valor % valor_nota
    print("Notas de R${:.2f}: {} nota(s)".format(valor_nota, qtdade_notas))
#-----------------------------------FIM---------------------------------
principal( )
